cico_build_deploy.sh

```diff
@@ -75,11 +75,11 @@ install_deps
 make build
 TAG=$(echo $GIT_COMMIT | cut -c1-${DEVSHIFT_TAG_LEN})
 if [[ ${TARGET:-"centos"} = 'rhel' ]]; then
-  docker build -t ${LOCAL_IMAGE_NAME} -f ./docker/Dockerfile.rhel . | cat
+  docker build -t ${LOCAL_IMAGE_NAME} -f ./docker/rhel.Dockerfile . | cat
```

Comment:
this file is obsolete now

Reply:
great! one less place to change :D ... but it's still there in the other k8s puller for rhche, FYI.

Reply:
for rh-che it is expected